### ISSUE TITLE
JSON error since update to 10.1.0...

### ISSUE THREAD

**Issue by skoontastic** [autopkg/filewave#5]
Autopkgr is sending errors on every run:

```
No JSON object could be decoded.
```

If I run a recipe, this is what I get:

```
autopkg run AdobeFlashPlayer.filewave
Processing AdobeFlashPlayer.filewave...
No JSON object could be decoded
Failed.

The following recipes failed:
    AdobeFlashPlayer.filewave
        Error in com.github.autopkg.filewave.FlashPlayerNoRepackage: Processor: com.github.autopkg.filewave.FWTool/FileWaveImporter: Error: No JSON object could be decoded

The following packages were copied:
    Pkg Path                                                                                                                   
    --------                                                                                                                   
    /Users/server/Library/AutoPkg/Cache/com.github.autopkg.filewave.FlashPlayerNoRepackage/AdobeFlashPlayer-20.0.0.267.pkg
```

I am on the aforementioned version 10.1.0 on the admin and server, and I have tried the latest stable and release candidate builds of Autopkgr. This seems to have started with 10.1.0.

**Comment by skoontastic** [comment]
Also, here is the traceback that comes with the error, if it helps:

```
Traceback (most recent call last): File "/usr/local/bin/autopkg", line 1459, in run_recipes autopackager.process(recipe) File "/Library/AutoPkg/autopkglib/__init__.py", line 479, in process % (identifier, step["Processor"], unicode(err))) AutoPackagerError: Error in com.github.autopkg.filewave.FlashPlayerNoRepackage: Processor: com.github.autopkg.filewave.FWTool/FileWaveImporter: Error: No JSON object could be decoded
```

**Comment by tempusthales** [comment]
I got that error on different files than the ones you used @skoontastic :

Can't create /Users/gpalau/Library/AutoPkg/Cache/com.github.autopkg.filewave.Silverlight/Silverlight/flatpkg: No such file or directory

No JSON object could be decoded

Could not find parent recipe for com.github.homebysix.pkg.Caffeine
No valid recipe found for com.github.homebysix.pkg.Caffeine

So maybe its an error caused by parent recipe being broken?

**Comment by tempusthales** [comment]
@johncclayton Following up on @skoontastic 

Pretty much every .filewave recipe explodes with and error similar to this one:

com.github.autopkg.filewave.Evernote
       Error in com.github.autopkg.filewave.Evernote: Processor: com.github.autopkg.filewave.FWTool/FileWaveImporter: Error: No JSON object could be decoded

Like @skoontastic mentioned since 10.1.0 the processor is not working... please advise.

**Comment by johncclayton** [comment]
I'll get back you ASAP on this (1:02am here at the mo, so not right now:-).  I believe it's already fixed in the upcoming 10.1.1 but will confirm.

**Comment by johncclayton** [comment]
Taking a look at this now...

**Comment by johncclayton** [comment]
Solved - coming to a 10.1.1 release (and v11 when it goes out the door) very very soon.
